src/chain_spec.rs

```diff
@@ -134,20 +129,7 @@ impl Alternative {
                             get_authority_keys_from_seed("Bob"),
                         ],
                         get_account_id_from_seed::<sr25519::Public>("Alice"),
-                        vec![
```

Comment:
Did we just moved them around or something changed here ?

Reply:
Yes, we have a spreadsheet, which is pinned to our #crcl_flim on Slack.

Reply:
These were moved here: https://github.com/centrifuge/centrifuge-chain/pull/144/files/3710754192fe895eb86de34533692a05e2511cec#diff-6740deafebf0ba9153f241785add2eefR240